metricbeat/module/kubernetes/util/kubernetes_test.go

```diff
@@ -265,13 +265,20 @@ func TestCreateMetaGenSpecific(t *testing.T) {
 	require.NoError(t, err)
 
 	log := logp.NewLogger("test")
+
+	namespaceConfig, err := conf.NewConfigFrom(map[string]interface{}{
+		"enabled": true,
```

Comment:
is it needed to set explicitly? by default it is already `true`, no?

Reply:
By default is true, yes, but in this case we need to at it explicitly because the config is not unpacking the default one, like we now are doing for pod_test and service_test.